src/components/DependencyGraph.tsx

```diff
@@ -64,15 +65,24 @@ export const DependencyGraph: React.FC = () => {
     return (
         <>
             <div className={optionsCls}>
-                <h1 className={headerCls}>Judge-Dredd UI</h1>
-                {process.env.NODE_ENV === 'development' && (
-                    <Radio onChange={handleViewSwitchChange} toggle checked={areOnlyConnectedNodesShown} />
-                )}
-                <EnvironmentSelect disabled={isPending} options={environments} env={env} onEnvironmentChange={setEnv} />
+                <Button icon onClick={() => setIsMenuOpen(true)}>
```

Comment:
I think we should go for consistency in callback convention - either all of them should be defined as function property inside the class or passed as anonymous function directly in the component props.

Reply:
I use inline functions for basic setters (imo it is way clearer than defining a named function). Whenever there's more logic than one self-describing statement, I usually create a named function so that it gives information about what happens inside of it.

If you feel like we should standardize it better, I'm open for suggestions and able to change it right away